client/src/components/DataFiles/DataFilesListing/DataFilesListing.js

```diff
@@ -123,16 +122,15 @@ const DataFilesListing = ({ api, scheme, system, path, isPublic }) => {
   }, [api, showViewPath, fileNavCellCallback]);
 
   return (
-    <div styleName="root">
+    <>
       {!isPublic && (
         <DataFilesSearchbar
           api={api}
           scheme={scheme}
           system={system}
-          styleName="searchbar"
```

Comment:
__Explanation__
This class does not exist. No CSS Modules stylesheet exists in which it even could exists.